### DISCUSSION TITLE
Chia GUI does not show new NFT

### DISCUSSION THREAD

**Discussion by ddangdd** [Chia-Network/chia-blockchain#8208] [Support]
I have created multiple PLot NFT's in Chia Gui and there are a few not showing up in the UI. I have went ahead and tried to create more through command line and it seems like the max number of wallets is set at 20? if I need to create more Plot NFT's would I have to create another wallet?

Thank You.

**Comment by loppefaaret** [comment]
A future update will allow more plotNFT's to be listed - for now i believe the limit is around 18-20 ? 
it should be able to manage them through the CLI on version 1.2.3, until we get 1.2.4 out with a GUI fix for this issue

**Comment by BrandtH22** [comment]
@ddangdd , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
